### DISCUSSION TITLE
Error - Capstonne Web scraping with workcloud pada app flask

### DISCUSSION THREAD

**Discussion by ArijayaAzis** [teamalgoritma/community#916] [Common Error]
Karena menggunakan WorkCloud, jadi mengubah 'data' (pada line 74) menjadi 'wordcloud2', dan menghasilkan output error
![image](https://github.com/user-attachments/assets/862e4838-ff38-4c2c-8411-1d418442347d)
<img width="920" alt="image" src="https://github.com/user-attachments/assets/c6b7da14-fd9c-48d9-8564-cf6317c118cc">

**Comment by saskia-dwi-ulfah** [comment]
Dear @ArijayaAzis 

Terima kasih  sudah bertanya melalui Github community Algoritma. Dari kode yang sudah dilampirkan,kita perlu memodifikasi cara plot wordcloud agar dapat ditampilkan dengan benar, pak. Bisa dicoba dengan kode di bawah ini.

```
	card_data = f'' #be careful with the " and ' 

	# generate plot
	fig, ax = plt.subplots(figsize=(10, 5)) # Display the WordCloud on the axis 
	ax.imshow(wordcloud, interpolation='bilinear') 
	ax.axis('off') # Remove axes
```
Di sini kita mendefine axisi baru dan menggunakan fungsi `.imshow()` untuk menampilkan wordcloud yang sudah di buat. Untuk `card_data` bisa diisi saja dengan string yang kosong `''`.

Semoga jawaban ini dapat membantu.

Best regards,
Team Algoritma

**Comment by adizaen** [comment]
Dear @ArijayaAzis 

Terima kasih telah bertanya melalui Github discussions Algoritma. Untuk menjaga agar forum diskusi berjalan kondusif, karena tidak ada respon lanjutan setelah 7 hari dari respon terakhir, kami izin menandai pertanyaan sudah terjawab dan menutup diskusi ini. Semoga dapat dipahami dan terima kasih.

**Best regards,**
Team Algoritma
